wsdl/ver20/media/wsdl/media.wsdl

```diff
@@ -1300,6 +1338,257 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:complexType>
 			</xs:element>
 			<!--===============================-->
+			<!--============AudioClipManagement===================-->
+			<xs:complexType name="AudioClip">
+				<xs:sequence>					
+					<xs:element name="Enabled" type="xs:boolean">
```

Comment:
What does Enabled parameter actually do? What happens if a Client sends Play for disabled Audio Clip? (note: we don't have any error about this action for Play)

Reply:
Enable parameter  control the audio clip playback operation. If Enable is false, the clip is disabled for the playback operation. Yes, we will add error code for same in PlayAudioClip operation.